truss/build/configure.py

```diff
@@ -0,0 +1,63 @@
+import json
+import logging
+import os
+import sys
+from pathlib import Path
+from typing import Optional
+
+import typer
+from truss import load
+from truss.patch.hash import directory_content_hash
+from truss.patch.signature import calc_truss_signature
+from truss.patch.truss_dir_patch_applier import TrussDirPatchApplier
+from truss.server.control.patch.types import Patch
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+app = typer.Typer()
+
+
+@app.command()
+def configure_truss_for_build(
+    truss_dir: str,
+    build_context_path: str,
+    use_control_server: bool = False,
+    patches_path: Optional[str] = None,
+    hash_file_path: Optional[str] = None,
+    signature_file_path: Optional[str] = None,
+):
+    tr = load(truss_dir)
+
+    if patches_path is not None:
+        logging.info("Applying patches")
+        logger = logging.getLogger("patch_applier")
```

Comment:
I think to make this clearer & to be more explicit, let's define 2 loggers at the top-level in this file, and pass the patch_applier_logger separately (name them more clearly too)

Reply:
What's the general purpose of having multiple loggers? To use different log levels or output(files) for different loggers?

What about having just one logger and adding the file and log-location to the log format (absl-style logging)? It seems simpler to manage to me.

Reply:
I'll iterate on this in upcoming PRs as it becomes relevant. For now, I think it makes sense to not add too much refactoring to the TrussDirPatchApplier in this PR.